src/components/Board/Board.css

```diff
@@ -92,3 +124,115 @@
   overflow-x: hidden;
   flex: 1 1;
 }
+
+.Board__output.Board__output--expanded .SymbolOutput__value {
+  min-width: 20vh;
```

Comment:
The ideal will be to use % instead of vh . If not i think vdh can work in mobiles browsers